webpack.config.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+
+module.exports = {
+  mode: 'production',
+  context: path.resolve(__dirname, 'rt_dashboard'),
+  entry: {
+    app: ['./javascript/app.js']
+  },
+  output: {
+    path: path.resolve(__dirname, 'rt_dashboard/static'),
+    filename: 'js/[name].js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.js?$/,
+        exclude: /node_modules\//,
+        loader: 'babel-loader',
+        options: {
+          presets: ['@babel/preset-env', '@babel/preset-react']
+        }
+      },
+      {
+        test: /\.html$/i,
+        loader: 'html-loader',
+        options: {
+          attributes: {
+            list: [{
+              tag: 'link',
+              attribute: 'href',
+              type: 'src',
+              filter: (tag, attribute, attributes) => {
+                return false
+              }
+            }]
+          }
+        }
+      },
+      {
+        test: /\.s[ac]ss$/i,
+        use: [
+          MiniCssExtractPlugin.loader,
+          'css-loader',
+          'sass-loader'
+        ]
+      },
+      {
+        test: /\.(jpg|jpeg|png|gif|svg|eot|ttf|woff|woff2)$/i,
+        use: [{
+          loader: 'file-loader',
+          options: {
+            name: '[name].[ext]',
+            outputPath: 'css/fonts/',
+            publicPath: 'fonts'
+          }
+        }]
+      }
+    ]
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: 'css/[name].css'
+    })
+  ],
+  devtool: 'source-map',
+  target: 'web',
+  devServer: {
+    before: (app, server, compiler) => {
+      app.get('/admin/rt_dashboard/inner/queues.json', (req, res) => {
+        res.json({
+          queues: [
+            { name: '[test]', count: 0, url: '/admin/rt_dashboard/inner/test' },
+            { name: '[running]', count: 1, url: '/admin/rt_dashboard/inner/running' }
+          ]
+        })
+      })
+      app.get('/admin/rt_dashboard/inner/workers.json', (req, res) => {
```

Comment:
Ideally, rt_dashboard should not have any dependencies on django-admin or the path on which it is deployed.

Reply:
agreed, this is not a real dependency on django-admin, just uses similar path structure, because it is defined in the dev [index.html](https://github.com/djangsters/rt-dashboard/pull/13/files#diff-12b975df8ea43a91a5eff2e7c8afbf88R10). That would be updated so jinja can inject the actual prefix.